openpype/pipeline/create/creator_plugins.py

```diff
@@ -224,7 +243,8 @@ def identifier(self):
 
         return self.family
 
-    @abstractproperty
+    @property
+    @abstractmethod
```

Comment:
Isn't this Python 3.3+ only? [src](https://stackoverflow.com/a/48710068)

Reply:
Yes, but this is not Python 2 breaking. I would not bend to Python 2 at this moment.